### ISSUE TITLE
LIBMTP prefix

### ISSUE THREAD

**Issue by mkmkme** [panuhorsmalahti/mtp#4]
In C, there are no packages, so you need to add such prefix as 'LIBMTP' to understand that you calling libmtp function.
But in node.js you have such feature as 'require'.
So why don't you use

```
mtp.DetectRawDevices
```

or

```
mtp.Detect_Raw_Devices
```

instead of

```
mtp.LIBMTP_Detect_Raw_Devices
```

?
It looks redundant, doesn't it?

**Comment by panuhorsmalahti** [comment]
Yeah, it's reduntant. However, it retains a one to one correspondence with C functions, probably making it easier to use the library.
